conf/alluxio-env.sh.template

```diff
@@ -72,6 +72,22 @@
 # E.g. "-Dalluxio.user.file.writetype.default=CACHE_THROUGH"
 # ALLUXIO_USER_JAVA_OPTS
 
+# Config properties set for Alluxio shell. (Default: "-Xms256M -Xmx256M")
```

Comment:
Can we give more detailed description for when this opts is needed to be set?
e.g. # Config properties for the monitoring service used by Alluxio shell to monitor the healthiness of Alluxio master process.